comments/https_for_s3_static_websites.md

```diff
@@ -0,0 +1,86 @@
+
+![Start of CloudFront distribution setup options](images/https_for_s3_static_websites_viewer_policy.png)
+
+- Underneath *Viewer*, the "Viewer protocol policy" should be either "Redirect HTTP to HTTPS" or "HTTPS only". We're not going to the trouble of setting up HTTPS here just to allow HTTP connections anyway.
+- Underneath *Web Application Firewall*, pick the appropriate setting depending on how paranoid you are. Since this is a static website I'm of the strong opinion that you probably don't need to pay extra for a WAF.
+- Under *Alternate domain name (CNAME)*, enter the custom domain you want to use, e.g. `subdomain.example.com`.
+- Under *Custom SSL certficate*, open the dropdown. The certficate you requested earlier via Certificate Manager should be listed here. Select it.
+
+Once finished, click "Create distribution" at the bottom of the page.
+
+The distribution will take a while to create. When it's created, select it from the list of distributions to bring up the origin details and click the *Origins* tab. Select the S3 bucket origin and then "Edit" to bring up the origin details. Underneath *Origin access* there should now be a blue box telling you to give CloudFront permission to access the S3 bucket by adding a policy to the S3 bucket permissions.
+
+![Start of CloudFront distribution setup options](images/https_for_s3_static_websites_cloudfront_policy.png)
+
+Click the "Copy policy" button inside this blue box, and then "Go to S3 bucket permissions". Open the *Permissions* tab, and paste the policy info into the *Bucket policy* section here, editing as appropriate to make it co-exist nicely with any existing policies that you want to keep.
+
+Go back to CloudFront, and select your distribution again to view the details. Underneath *Distribution domain name* there should be a domain name that starts with a UID and looks like `abcde12345.cloudfront.net`. This is the CloudFront endpoint that it serves your bucket content under - so, if you have an `index.html` file and associated website content at `s3://bucket-name/directory/subdirectory/index.html`, then entering `https://abcde12345.cloudfront.net/directory/subdirectory` into your browser should load the website.
+
+### Route 53 (or other DNS)
+
+The final step is to add a DNS record for the custom domain aliasing it so that it points to the CloudFront distribution domain name. Route 53 has fancy dropdowns for this if you select the "Alias" toggle that let you connect to the CloudFront distribution, but ultimately this is just an A record with your custom domain `subdomain.example.com` as the record name and the CloudFront distribution domain name `abcde12345.CloudFront.net` as the record value.
```

Comment:
Could you add a screenshot for the `Alias` toggle please

Reply:
Difficult to get a good one, unfortunately.